### PR TITLE
Add about screen

### PR DESCRIPTION
**Pull request by lukstbit**
This PR implements the UI for the about screen(issue #33 ):
 - my implementation uses static text and code defined urls
 - there are no icons(I used the link material icon) for social platforms. We need to use vector assets to make them look good.

### How has it been tested?

Manually.

### CODE REVIEW

File: app/src/main/java/ro/code4/deurgenta/ui/about/AboutFragment.kt [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        super.onResume()
+        requireActivity().title = getString(R.string.about_title)
+    }
+
+    companion object {
+        const val CODE4RO_FACEBOOK = "https://www.facebook.com/code4romania/"
+        const val CODE4RO_INSTAGRAM = "https://www.instagram.com/code4romania"
+        const val CODE4RO_URL = "https://code4.ro/en"
+        const val CODE4RO_GITHUB = "https://github.com/code4romania"
+        const val CODE4RO_DONATE = "https://code4.ro/en/donate"
```
Comment (aniri) [inline app/src/main/java/ro/code4/deurgenta/ui/about/AboutFragment.kt]: could you please extract these in the properties files?

File: app/src/main/res/values-ro-rRO/strings.xml [hunk 2]
```diff
@@ -99,4 +99,17 @@
     <string name="edit_backpack_item_error_quantity">Introdu cantitatea produsului!</string>
     <string name="edit_backpack_item_error_quantity_value">"Introdu un numar pentru cantitate!"</string>
 
+    <!-- About -->
+    <string name="about_title">Despre aplicație</string>
+    <string name="about_app_description">Aceasta este descrierea acestei aplicații foarte importante. În mod normal
+        descrierea este mult mai lungă!</string>
```
Comment (aniri) [inline app/src/main/res/values-ro-rRO/strings.xml:149]: loving this! 🤣

**Comment by aniri** [comment]
a rebase is also needed 🙏

File: app/src/main/java/ro/code4/deurgenta/ui/about/AboutFragment.kt (hunk 1 above)
Comment (lukstbit) [reply app/src/main/java/ro/code4/deurgenta/ui/about/AboutFragment.kt]: Done!

**Comment by lukstbit** [comment]
I rebased, the PR can be merged cleanly.

**Review by aniri — APPROVED** [review]

**Pull request merged**